bots/wallet/bot.js

```diff
@@ -5,7 +5,7 @@ function calculateFee(n, tx) {
     }
 
     var gasMultiplicator = Math.pow(1.4, n).toFixed(3);
-    return web3.fromWei(web3.eth.gasPrice * gasMultiplicator * estimatedGas, "ether");
+    return parseFloat(web3.fromWei(web3.eth.gasPrice * gasMultiplicator * estimatedGas, "ether")).toFixed(7);
```

Comment:
No, parseFloat actually [parses numbers perfectly well](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseFloat) (I can see that in french version of mozilla js doc, the signature reads `parseFloat(string)`, while in default english one, it's `parseFloat(value)`, very weird :))
The problem with `fromWei` is that sometimes it returns string, sometimes it's BigNumber (in my local testing, it seems to always return string the way how it's used in wallet bot), anyway I find the choice of multiple return type very poor and only complicating thing, running it through `parseFloat` makes sure it's number on which we can call `.toFixed(n)` method.